src/lib/gw2/itemStats.js

```diff
@@ -0,0 +1,90 @@
+import { queryItemAttributes } from 'gw2itemstats';
```

Comment:
can we put this behind a lazy loaded chunk? we're currently using `promise-loader`, see: https://github.com/gaearon/promise-loader

just so its not downloaded until its needed ;-)